tests/integration/interactive/cmd_init_test.go

```diff
@@ -0,0 +1,69 @@
+package interactive
+
+import (
+	"bytes"
+	"fmt"
+
+	"github.com/Netflix/go-expect"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/redhat-developer/odo/tests/helper"
+)
+
+var _ = Describe("odo login and logout command tests", func() {
+
+	var commonVar helper.CommonVar
+
+	// This is run before every Spec (It)
+	var _ = BeforeEach(func() {
+		commonVar = helper.CommonBeforeEach()
+	})
+
+	// // Clean up after the test
+	// // This is run after every Spec (It)
+	var _ = AfterEach(func() {
+		helper.CommonAfterEach(commonVar)
+	})
+	It("should download correct devfile", func() {
+
+		Command := []string{"odo", "init"}
+		output, err := helper.RunInteractive(commonVar, Command, func(c *expect.Console, output *bytes.Buffer) (bytes.Buffer, error) {
```

Comment:
Instead of passing output as a bytes.Buffer, I think you can pass it as a io.Writer, so the test function is limited to writing into it.